src/main/java/com/thebois/models/Position.java

```diff
@@ -42,6 +44,23 @@ public void setPosY(final float posY) {
         this.posY = posY;
     }
 
+    @Override
+    public boolean equals(final Object o) {
```

Comment:
I remember we had a discussion if this equals method actually returned the correct value. Have we made sure it does?

Reply:
I believe it should since 2 positions are equal if they have the same x and y value.

Reply:
Thought for some reason we already had those tests, but apparently not. I'll add them.